cloudahoywriter.lua

```diff
@@ -6,29 +6,59 @@
 -- Code conventions
 --  - Use local when possible
 --  - Globals are prefixed with CAWR_
+--  - Method names use_underscores
+--  - Variable names useCamelCase
+--  - Constant names are CAPITAL_WITH_UNDERSCORE
 -----------------------------------------------
-local versionNum = '0.0.1'
+local versionNum = '0.0.2'
 
 require('graphics')
 
 -------------------- CONSTANTS --------------------
 local LOG_INTERVAL_SECS = 0.3
 local SECONDS_PER_MINUTE = 60
 local SECONDS_PER_HOUR = SECONDS_PER_MINUTE * 60
+local NOTIFICATION_SECS = 10
```

Comment:
Could you add a comment explaining what notification this is?

Reply:
Done